README.md

```diff
@@ -306,7 +306,17 @@ https://medium.com/@joehonton/cipher-suites-demystified-ada2e97be9c9
 
 ## Development
 
-* Ensure Python 3.8 is present along with venv and dev header files;
+### Setup
+* Ensure Python 3.9 is present along with venv and dev header files;
 * Setup virtual environment: `python3 -m venv keripy`
 * Activate virtual environment: `source keripy/bin/activate`
 * Setup dependencies: `pip install -r requirements.txt`
+
+### Testing
+* Install a testing framework like pytest: `pip install pytest`
+* To run the test suite: 
+    ```
+    pytest tests/ --ignore tests/demo/
```

Comment:
I had to do this to fix some test isolation issues.  Without this the test_ogling.py tests fail.